CodeStarterCamp_Week1/main.swift

```diff
@@ -3,91 +3,83 @@ import Foundation
 drawIceTop()
 drawIceBar()
 
+func drawPeperoBarLength(BarLength: Int) {
+    for _ in 1...BarLength {
+        print(" | | ")
+    }
+}
+
+func drawPeperoBody(body: String, topping: String) {
+    print(topping + body + topping)
+}
+
+func drawPeperoBar(lenght: Int, topping: String, body: String) {
+    if body == "***" {
+        for _ in 1...lenght {
+            drawPeperoBody(body: body, topping: topping)
+        }
+    }
+    else {
```

Comment:
```suggestion
    } else {
```
if-else 에서 else 는 보통 if 문의 닫는 괄호 뒤에다가 붙여서 작성합니다 🙂